standards/out-of-hours.md

```diff
@@ -0,0 +1,61 @@
+
+There are also cases where services may need out of hours support, but we're not currently able to provide it without risking the health and well-being of our staff, usually because we don't have enough staff who are able to support it. In these cases, we should either engage a wider group of staff to support a set of services, or seek the help of suppliers to provide sustainable support.
+
+## Determining if your service needs support out of hours
+
+Not all services need out of hours support. When considering the need for out of hours support for your service, you first need to understand the risk if the service were to go down when no-one is around to fix it.
+
+The kinds of risk that might suggest out of hours support is necessary include:
+
+- Risk to someone's life
+- The MOJ's ability to provide critical, time-sensitive justice services
+- Impact to staff's and suppliers' ability to do carry out their job safely and to the best of their ability
+- Impairing citizens', suppliers', and partners' ability to interact with MOJ services in a way that meets their needs
+
+These kinds of risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (realistically this means at least six people, ideally eight), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
```

Comment:
This language feels patronising to me. I have asked to be taken off an on call rota before because of the extra burden it placed on my mental health. Framing it as "not integrating being on call into your lives" portrays it as personal failing rather than recognising that people have different lives and experience on call differently.

Reply:
Thanks Mat, it's certainly not my intention for this to be patronising. How about something like this instead:

> Being on call can be stressful, and can impact people's mental health and personal lives if not managed well. If that happens, people will withdraw from the rota (leaving the service unsupported).
>
> To avoid that, a few core criteria must be met:

Reply:
Yup, I think so, message is much clearer now 👍